synapseclient/models/file.py

```diff
@@ -21,14 +21,49 @@
 
 @dataclass()
 class File:
+    """A file within Synapse.
+
+    Attributes:
```

Comment:
Due to https://github.com/microsoft/pylance-release/discussions/4759 pylance does not support seeing attributes in the doc string and linking them in the hover document. You'll see in these files that the information is duplicated.

Reply:
Ahhh, I see.  That's really unfortunate...